### PR TITLE
Task/2.4 test

### PR DESCRIPTION
**Pull request by afoerster**

### CODE REVIEW

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/JsonSupport.scala [hunk 1]
```diff
@@ -1,19 +1,4 @@
-/*
```
Comment (bpmcd) [inline log-collector/src/main/scala/io/phdata/pulse/logcollector/JsonSupport.scala:1]: Don't you still want the license header here?

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/Stream.scala [hunk 2]
```diff
@@ -74,35 +74,8 @@ abstract class Stream[EventType](streamParams: StreamParams = StreamParams()) ex
...
-          case _ => throw new IllegalArgumentException(s"Unrecogized overflow strategy $strategy")
-        }
-      }
-      .getOrElse {
-        failOverflowStrategy
-      }
-
   logger.info(s"Max buffer size `pulse.collector.stream.buffer.max` is $maxBuffersize")
-  logger.info(s"Overflow stragegy `pulse.collector.stream.overflow.strategy` ${overflowStrategy}")
+  //logger.info(s"Overflow stragegy `pulse.collector.stream.overflow.strategy` ${overflowStrategy}")
```
Comment (bpmcd) [inline log-collector/src/main/scala/io/phdata/pulse/logcollector/Stream.scala:78]: Delete?

File: log-collector/src/main/scala/io/phdata/pulse/logcollector/Stream.scala [hunk 3]
```diff
@@ -155,14 +128,15 @@ abstract class Stream[EventType](streamParams: StreamParams = StreamParams()) ex
...
+        .mapParallelUnordered(numThreads) { y =>
           Task(save(group.key, y.map(_._2))).onErrorRecover {
             case e: Throwable =>
               logger.error(s"Exception writing batch for application ${group.key}", e)
-        })
-        .executeOn(blockingScheduler)
-        .foreach(_ => ())
+          }
+        }
+        .foreach(_ => ());
```
Comment (bpmcd) [inline log-collector/src/main/scala/io/phdata/pulse/logcollector/Stream.scala:137]: semi-colon?

File: log-collector/src/test/scala/io/phdata/pulse/logcollector/SolrCloudStreamTest.scala [hunk 4]
```diff
@@ -90,7 +90,7 @@ class SolrCloudStreamTest extends FunSuite with BaseSolrCloudTest {
     assert(queryResult.getResults.getNumFound > 0)
   }
 
-  test("Stream failure method called if it exceeds the maximum queue size") {
+  ignore("Stream failure method called if it exceeds the maximum queue size") {
```
Comment (bpmcd) [inline log-collector/src/test/scala/io/phdata/pulse/logcollector/SolrCloudStreamTest.scala:93]: Do you intend to re-enable this test?  If not, just delete it.

**Comment by afoerster** [comment]
This PR was not intended to be merged, closing

**Pull request closed without merging**